.github/release.yml

```diff
@@ -0,0 +1,22 @@
+changelog:
+  exclude:
+    labels:
+      - skip-changelog
+      - documentation
+      - ci
```

Comment:
In the spirit of producing more information, should we leave these two tags out? I'm thinking of several scenarios where we maybe make some critical improvements to the CI / build process, and it's worth documenting those down as well.

Including the current documentation efforts (linting and what-not), those PRs get tagged as "Documentation", but I think it's better to keep them in the changelog (condensed as 1 "Change" of course, not as individual items).

Reply:
Good point. We could move these to a new section e.g. "documentation and maintenance", which we can then manually scan and either delete entirely, or cherry-pick anything notable